api/shape/getTags.js

```diff
@@ -1,8 +1,9 @@
+/*
 import Shape from './Shape.js';
 
 // This should take an op.
 
-export const getTags = Shape.registerMethod2(
+export const getTags = Shape.registerMethodX(
```

Comment:
What is registerMethodX?

Reply:
Ah, just something so that the commented out code didn't match registerMethod2 anymore -- I'll remove that.